packages/eslint-plugin-calcite-components/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "ignorePatterns": ["*"]
```

Comment:
Any package with this configuration will skip linting entirely, right? If so, won't this make the contributing instructions on linting misleading?

Reply:
For now, maybe the contributing doc can be updated to indicate that ESLint will only run on `calcite-components` and we can tweak our linting config to lint properly outside `calcite-components` after this lands?

Reply:
Yeah I added this to CCR and the eslint plugin so eslint won't complain about there not being a config which covers their files.  

I'm working on pulling shared eslint/tsconfigs into separate workspaces as part of the lint-staged fix. After that happens, the base eslint config will be used in these packages. 

So I'm going to leave the doc as is for now so I don't have to change it and then change it back.